stats/stats.go

```diff
@@ -36,7 +36,10 @@ type RPCStats interface {
 	IsClient() bool
 }
 
-// Begin contains stats when an RPC attempt begins.
+// Begin contains stats when an RPC attempt begins. This event is called after
+// the InHeader event, as headers must be processed before the RPC lifecycle
```

Comment:
nit `InHeader`

Reply:
What specifically is the issue here? Is it related to style?